src/JMS/Serializer/Metadata/Driver/AnnotationDriver.php

```diff
@@ -158,8 +158,17 @@ public function loadMetadataForClass(\ReflectionClass $class)
                         $propertyMetadata->serializedName = $annot->name;
                     } elseif ($annot instanceof Expose) {
                         $isExpose = true;
+                        if (null !== $annot->if) {
+                            $propertyMetadata->excludeIf = "!(" . $annot->if . ")";
+                            $classMetadata->usingExpression = true;
```

Comment:
Better move this to the ``addPropertyMetadata`` method, then we do not have to check this in each driver.